README.md

```diff
@@ -1,3 +1,3 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+In order to run this app, you must add .evn-development and .env-test files in the main directory. These files should include a PGDATABASE and a PGPASSWORD for sql. The databases should be called nc_news and nc_news_test respectively.
```

Comment:
This needs to be clearer as to what needs to be put in those respective files. PGPASSWORD is also only for users who are not using Mac, so if you are going to mention this, I would be more specific about that also.